test/eth/TransactionObject.spec.js

```diff
@@ -10,9 +10,9 @@ import { ETH, WETH } from '../../src/eth/Currency';
 
 let service;
 
-beforeAll(() => {
+beforeAll(async () => {
   service = buildTestEthereumTokenService();
-  return service.manager().authenticate();
+  await service.manager().authenticate();
```

Comment:
no need to change this back, but FYI these are identical -- the jest hooks will automatically await if the function returns a promise